### PR TITLE
[11.x] Fix dockblock `AnonymousEvent` class

### PR DESCRIPTION
**Pull request by michaelnabil230**
This PR is for fixing the dock block in `AnonymousEvent` class

### CODE REVIEW

**Comment by crynobone** [comment]
CleanShot 2024-04-18 at 10 26 49

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 1]
```diff
@@ -13,32 +13,43 @@ class AnonymousEvent implements ShouldBroadcast
 
     /**
      * The connection the event should be broadcast on.
+     *
+     * @var string|null
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:17]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 2]
```diff
@@ -13,32 +13,43 @@ class AnonymousEvent implements ShouldBroadcast
...
      */
     protected ?string $connection = null;
 
     /**
      * The name the event should be broadcast as.
+     *
+     * @var string|null
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:24]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 3]
```diff
@@ -13,32 +13,43 @@ class AnonymousEvent implements ShouldBroadcast
...
      * The payload the event should be broadcast with.
+     *
+     * @var array<string, mixed>
      */
     protected array $payload = [];
 
     /**
      * Should the broadcast include the current user.
+     *
+     * @var bool
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:38]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 4]
```diff
@@ -13,32 +13,43 @@ class AnonymousEvent implements ShouldBroadcast
...
      */
     protected bool $includeCurrentUser = true;
 
     /**
      * Indicates if the event should be broadcast synchronously.
+     *
+     * @var bool
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:45]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 5]
```diff
@@ -13,32 +13,43 @@ class AnonymousEvent implements ShouldBroadcast
...
      */
     protected bool $shouldBroadcastNow = false;
 
     /**
      * Create a new anonymous broadcastable event instance.
      *
+     * @param  \Illuminate\Broadcasting\Channel|array|string  $channels
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:52]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 6]
```diff
@@ -48,6 +59,9 @@ public function __construct(protected Channel|array|string $channels)
 
     /**
      * Set the connection the event should be broadcast on.
+     *
+     * @param  string  $connection
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:63]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 7]
```diff
@@ -58,6 +72,9 @@ public function via(string $connection): static
 
     /**
      * Set the name the event should be broadcast as.
+     *
+     * @param  string  $name
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:76]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 8]
```diff
@@ -68,6 +85,9 @@ public function as(string $name): static
 
     /**
      * Set the payload the event should be broadcast with.
+     *
+     * @param  \Illuminate\Contracts\Support\Arrayable|array  $payload
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:89]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 9]
```diff
@@ -92,6 +114,8 @@ public function toOthers(): static
 
     /**
      * Broadcast the event.
+     *
+     * @return void
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:118]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 10]
```diff
@@ -102,6 +126,8 @@ public function sendNow(): void
...
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:130]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 11]
```diff
@@ -114,6 +140,8 @@ public function send(): void
 
     /**
      * Get the name the event should broadcast as.
+     *
+     * @return string
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:144]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/AnonymousEvent.php [hunk 12]
```diff
@@ -142,6 +170,8 @@ public function broadcastOn(): Channel|array
 
     /**
      * Determine if the event should be broadcast synchronously.
+     *
+     * @return bool
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/AnonymousEvent.php:174]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/BroadcastManager.php [hunk 13]
```diff
@@ -134,6 +134,9 @@ public function socket($request = null)
 
     /**
      * Begin sending an anonymous broadcast to the given channels.
+     *
+     * @param  \Illuminate\Support\Channel|string|array  $channels
+     * @return \Illuminate\Broadcasting\AnonymousEvent
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/BroadcastManager.php:139]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/BroadcastManager.php [hunk 14]
```diff
@@ -150,6 +156,9 @@ public function private(string $channel): AnonymousEvent
 
     /**
      * Begin sending an anonymous broadcast to the given presence channels.
+     *
+     * @param  string  $channel
+     * @return \Illuminate\Broadcasting\AnonymousEvent
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/BroadcastManager.php:161]: not needed as described in the documentation.

File: src/Illuminate/Broadcasting/BroadcastManager.php [hunk 15]
```diff
@@ -142,6 +145,9 @@ public function on(Channel|string|array $channels): AnonymousEvent
 
     /**
      * Begin sending an anonymous broadcast to the given private channels.
+     *
+     * @param  string  $channel
+     * @return \Illuminate\Broadcasting\AnonymousEvent
```
Comment (crynobone) [inline src/Illuminate/Broadcasting/BroadcastManager.php:150]: not needed as described in the documentation.

**Comment by driesvints** [comment]
Thanks but it's fine.

**Pull request closed without merging**